src/maps.json

```diff
@@ -25,5 +25,41 @@
     "ground": "Gras",
     "public_transport": "S1/S11, Bus 37/283, Haltestelle S Bahrenfeld",
     "coordinates": [53.560679, 9.902605]
+  },
+  {
+    "id": "4",
```

Comment:
Wieso ist die id ein String?

Reply:
Warum auch immer, werden die Anführungszeichen ständig automatisch gesetzt.. Ich hab mich jetzt erstmal nicht weiter darum gekümmert, weil ich die Daten eh bald in eine Datenbank übertrage